spec/stud/interval_spec.rb

```diff
@@ -1,8 +1,19 @@
 require "stud/interval"
 require "spec_env" # from the top level spec/ directory
 
+module Stud
+  def reset!(target = Thread.current)
```

Comment:
"expose it" meant "expose the method as a public api", in other words, move the `reset!` method in `Stud` just list `stop!`. 

does this make more sense?